src/components/elements/button/button.scss

```diff
@@ -41,6 +23,13 @@
     svg path {
       fill: #ffffff;
     }
+    &.disabled, &:disabled {
```

Comment:
Please create a dedicated story for the disable mode changes, so it will be possible to see that it fits Anna's design. The story should include all variants and their correlating disable mode.

Reply:
I added it to the button with icon story, should it be a separate story or is that good enough?

Reply:
good enough